intro/setup/index.html

```diff
@@ -0,0 +1,349 @@
+                <li>Familiarity with basic OOP(Object Oriented Programming)</li>
+                <li>Basic control flow keywords such as - for, while, break, if, elif, else</li>
+                <li>Functions</li>
+            </ul>
+        
+        Since in Python everything is an object, that is also how you would 
+        interact with the utilities provided through pygame, so knowing how they work 
+        is crucial.
+    </p>
+
+    <h2><a id="installation" class="js"></a>Installing pygame</h2>
+    <p>
+        pygame can be installed through <a href="https://pypi.org">pypi</a> by
+        using the pip package manager.
+
+        This is as simple as,
+    </p>
+
+    <p>
+{% highlight python %}
+pip install pygame
+{% endhighlight %}
+
+    BUT, how you access your pip depends on your Operating System and what interpreter 
+    you plan on using to run the code you will be writing throughout this article. 
+
+    For the sake of this article, we recommend installing pygame globally, and using your global interpreter 
+    to run your code. 
+    
+    For windows, the way you access pip can be 
+
+
+{% highlight python %}
+pip install pygame 
+# Or
+
+# Accessing pip as a module through your python launcher
+py -m pip install pygame 
+
+python -m pip install pygame
+{% endhighlight %}
+
+    If you aren't too confident with where pygame is being installed, use your python launcher 
+    as shown in the last 2 methods to install it, and use the same launcher to run the code accordingly!
+
+    </p>
+
+    <h2><a id=""></a></h2>
+
+    <h2><a id="project-directory" class="js"></a>Setting up the project directory</h2>
+    <p>
+        Throughout this whole miniseries, you will be required to work with a couple different 
+        files, it would be better and recommended if everything required to run your game is within 
+        one directory. We will create the required folders as we go, rather than take the generic 
+        approach and dump everything into our project folder that we wouldn't need(yet). But, 
+        we do still need some basic setup.
+    </p>
+
+    <p>
+        First, create a folder whereever you would like, and name it whatever you want. I will be
+        naming it <b>pygame_platformer</b> and I will store it in my <b>d:\dev\projects\</b> directory.
+        Once you have your project directory created, make a <b>main.py</b> file. This is where 
+        the code for our game will be! For now that is enough. If you check the contents of your project
+        folder, it should look like this:
+{% highlight python %}
+PS D:\dev\projects\pygame_platformer> ls
+
+
+    Directory: D:\dev\projects\pygame_platformer
+
+
+Mode                 LastWriteTime         Length Name
+----                 -------------         ------ ----
+-a----        11-08-2022     22:09              0 main.py
+
+
+{% endhighlight %}
+    </p>
+
+    <h2><a id="opening-a-window" class="js"></a>Opening a window with pygame</h2>
```

Comment:
Do we want section headers to have the major parts capitalized (like titles)? My vote is yes

Reply:
Poke